compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenInterfaces.td

```diff
@@ -199,6 +199,14 @@ def IREECodegen_InnerTileDescAttrInterface :
         return failure();
       }]
     >,
+    InterfaceMethod<
```

Comment:
... This doesn't go here? It's not general to inner tiled operations?

If it does go here, give it semantics that don't mention MMA